db/migrations/20240613150913_category.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Add migration script here
+CREATE TABLE IF NOT EXISTS Category (
+    markdown_id UUID PRIMARY KEY REFERENCES Markdown(markdown_id),
```

Comment:
Not sure if I should make a `markdown_id` a `PRIMARY KEY` or not. Let me know what you think

Reply:
Ahh! I think we probably want a `category_id` here to be the primary key, I usually like having a specific id for each table

Also, we might change how we attach categories to `Markdowns` eventually, and I think it will be easier to do that if each has its own id as a primary key!

Reply:
awesome! For the category_id it is the generated UUID inside the `store_markdown`?

for example it will be something like this:

`async fn store_category(
) -> color_eyre::Result<Category> {
let category_id = //generate UUID =
}`